hw_21.09.23/task2.py

```diff
@@ -0,0 +1,73 @@
+class Library:
+
+    def __init__(self, name):
+        self.name = name
+        self.books = []
+        self.authors = []
+
+    def new_book(self, name: str, year: int, author: object):
```

Comment:
`author: Author` would indicate that this must be an instance of the Author class